.github/workflows/end-to-end-tests.yml

```diff
@@ -134,6 +134,14 @@ jobs:
             path: artifacts
             if-no-files-found: ignore
 
+      - name: Upload the build directory as GitHub artifact
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        if: always()
+        with:
+            name: wordpress-build
```

Comment:
Can we have more specific artifact names? If this is for each PR, we should probably append the PR number. Or we could be more specific and use commit SHA.